etl/steps/data/garden/artificial_intelligence/2025-04-18/cset.meta.yml

```diff
@@ -0,0 +1,218 @@
+
+  description_short_investment: Only includes private-market investment such as venture capital; excludes all investment in publicly traded companies, such as "Big Tech" firms. This data is expressed in US dollars, adjusted for inflation.
+  description_short_articles: English- and Chinese-language scholarly publications related to the development and application of AI. This includes journal articles, conference papers, repository publications (such as arXiv), books, and theses.
+  description_short_patents: Patents related to artificial intelligence first submitted in the selected country's patent office. Subsequent granting of that patent could be by any country's patent office.
+
+  description_short_citations: Citations take time to accumulate so recent years may have fewer citations, and the measure for these years may not completely reflective of their eventual total.
+
+  note_citations: The citation counts for papers published in a specific year are subject to a time lag, as papers often accrue most citations in subsequent years. Data for 2022-2023 is therefore incomplete.
+  note_articles:  Data for 2022-2023 is incomplete.
+  note_investment: Data is expressed in constant 2021 US$. Inflation adjustment is based on the US Consumer Price Index (CPI). Data for 2022-2023 is incomplete.
+  note_patents: According to calculations by CSET, the median time for a patent to be granted is 826 days from its initial filing date, while the average time is 860 days.  Data for 2021-2023 is incomplete.
+
+  description_key_articles: &description_key_articles |-
+    - CSET sources the data on the number of articles and citations from the Merged Academic Corpus (MAC), which includes over 260 million academic articles.
+    - Articles are marked as AI-related using an automated system.
+    - Articles are linked to countries based on the authors’ listed organizations—if at least one author is from an institution in a country, the article counts for that country.
```

Comment:
Maybe you can drop the `-` after listed organizations. It works just fine without it